### PR TITLE
css-three-column-layout - Jacob Husband

### PR DESCRIPTION
**Pull request by jacobhusband**
Screen Shot 2022-08-25 at 12 25 59 PM

### CODE REVIEW

File: css-three-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+.col {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.circle-img {
+  width: 100%;
+  background-color: white;
+  border-radius: 100%;
```
Comment (thebearingedge) [inline css-three-column-layout/styles.css:51]: To get an exact circle you can set width and height to px sizes and then use 50% for your border radius.

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.one-third {
+  flex-basis: calc(100%/3);
+}
+
+.one-half {
+  max-width: 40%;
+}
```
Comment (thebearingedge) [inline css-three-column-layout/styles.css:60]: This class name is a bit misleading

File: css-three-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+h3 {
+  font-size: 1.5rem;
+  margin: 2rem 0 1rem;
+}
+
+p {
+  margin: 1.5rem;
+  color: #68d5a7;
+}
```
Comment (thebearingedge) [inline css-three-column-layout/styles.css:31]: In the reference image the text is a white-ish color

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Nice! Just a couple small tweaks

**Comment by jacobhusband** [comment]
Screen Shot 2022-08-25 at 12 40 42 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**